tests/test_cli.py

```diff
@@ -218,6 +225,17 @@ def test_output_filename_spaces(self, sample_pdf: str) -> None:
         result = self.run_cli([sample_pdf, "--output-filename", output_filename])
         result.assert_success()
 
+    @pytest.mark.skipif(
+        platform.system() == "Darwin", reason="filename not supported on macOS"
```

Comment:
macOS does [not seem to support](https://github.com/freedomofpress/dangerzone/actions/runs/8645853053/job/23703882932#step:6:446) invalid UTF-8 sequences in filenames, so macOs is skipped.

Reply:
Nice catch, but I'd still like to test as many corner cases in macOS as possible. For this reason, I propose using the `uncommon_text` fixture here, and simply remove the invalid Unicode character. The rest of the tests should ideally work.

Also, this would be a good opportunity to merge it with a test we have for filenames with spaces in it (`test_output_filename_spaces`).